components/Sidebar/index.tsx

```diff
@@ -1,13 +1,12 @@
-import Link from 'next/link';
-import Router, { useRouter } from 'next/router';
```

Comment:
useRouter got deleted which is causing the build error

Reply:
ah - thanks!